frontend/packages/ceph-storage-plugin/src/components/kms-config/kms-config.tsx

```diff
@@ -137,143 +62,24 @@ export const KMSConfigure: React.FC<KMSConfigureProps> = ({
       >
         <FormSelect
           value={kmsProvider}
-          onChange={setKMSProvider}
+          onChange={setKMSProvider(dispatch)}
           id="kms-provider"
           name="kms-provider-name"
           aria-label={t('ceph-storage-plugin~kms-provider-name')}
-          isDisabled
+          isDisabled={!isOdf}
```

Comment:
need to add feature guard here, so that dropdown is disabled for ODF 4.9 + OCP 4.10 cases.

Reply:
better you add TODO in code

Reply:
added here: https://github.com/openshift/console/pull/10475